canonical-web-and-design/vanilla-framework
# Read class reference data from SCSS files

## Done

Based on work from the hackaton, we are reading Yaml data for class reference from the comments inside SCSS files.

## QA

- Open [demo](https://vanilla-framework-4349.demos.haus/)
- Check the components with class references:
  - https://vanilla-framework-4349.demos.haus/docs/base/code#class-reference
  - https://vanilla-framework-4349.demos.haus/docs/patterns/notification#class-reference
  - https://vanilla-framework-4349.demos.haus/docs/patterns/navigation#class-reference
- Pull branch locally
  - run `dotrun build-class-references`
  - make sure it builds without errors and creates build/classreferences.yaml file
